bedrock/settings/__init__.py

```diff
@@ -106,13 +106,35 @@
     INSTALLED_APPS.pop(INSTALLED_APPS.index("bedrock.redirects"))
     MIDDLEWARE.pop(MIDDLEWARE.index("bedrock.redirects.middleware.RedirectsMiddleware"))
 
-    # TODO: define in Pocket-appropriate versions of
-    # DEV_LANGUAGES, PROD_LANGUAGES, CANONICAL_LOCALES,
+    # Override the FLUENT_* config for Pocket mode
+
     FLUENT_DEFAULT_FILES = [
         "brands",
         "nav",
         "footer",
     ]
+
+    # Note that for Pocket mode, we don't need an intermediary mozmeao/* repo to hold the translations
+    # while we calculate activation metadata via a CI task (which does happen for Mozorg). Why? All
+    # locales in Pocket should be 100% ready to go, because they are translated by a vendor, whereas
+    # Mozorg has community translation contributions as well, which aren't always exhaustive.
+    #
+    # As a result, both FLUENT_REPO and FLUENT_L10N_TEAM_REPO point to the same git repo
+
+    # These are already pre-defined in settings.base, but we make the POCKET_ ones here the defaults
+    # so that L10N mechanics don't need any other code branching in them
+    FLUENT_REPO = POCKET_FLUENT_REPO
+    FLUENT_REPO_URL = POCKET_FLUENT_REPO_URL
+    FLUENT_REPO_BRANCH = POCKET_FLUENT_REPO_BRANCH
+    FLUENT_REPO_PATH = POCKET_FLUENT_REPO_PATH
+
+    FLUENT_L10N_TEAM_REPO = config("POCKET_FLUENT_L10N_TEAM_REPO", default="mozilla-l10n/pocket-www-l10n")
```

Comment:
This should always be the same as `POCKET_FLUENT_REPO` in `base.py`. So I think you can just use those values here and avoid defining them twice and using different env vars. i.e.

```python
FLUENT_L10N_TEAM_REPO = POCKET_FLUENT_REPO
FLUENT_L10N_TEAM_REPO_URL = POCKET_FLUENT_REPO_URL
FLUENT_L10N_TEAM_REPO_PATH = POCKET_FLUENT_REPO_PATH
FLUENT_L10N_TEAM_REPO_BRANCH = POCKET_FLUENT_REPO_BRANCH
```

Reply:
or even just add the `L10N_TEAM` ones to the definitions above.